src/runner/api/itemRunner.js

```diff
@@ -427,6 +427,19 @@ var itemRunnerFactory = function itemRunnerFactory(providerName, data, options)
             if (_.isFunction(provider.renderFeedbacks)) {
                 provider.renderFeedbacks.call(this, feedbacks, itemSession, done);
             }
+        },
+
+        /**
+         * Provide access the data about APIP the content and related media files
+         *
+         * @returns {Object} the item's APIP data section
+         */
+        getApipData: function getApipData() {
```

Comment:
This is an implementation-specific method. It should not be placed here.

The item runner API is agnostic. There is nothing directly related to QTI nor APIP in it.

If you want to extend the API with specific behavior or dedicated entry, you should do it on the provider. Like here: https://github.com/oat-sa/tao-test-runner-qti-fe/blob/master/src/provider/qti.js#L177

Or you may add a generic method. But in this case it must add a real benefit.

Reply:
You're right this method doesn't make sense since it's linked to QTI2. 
But multiple time we needed to retrieve the item runner's data without having to listen an event.

I suggest you to add the method 
```
getData() { }
```
that will return the whole item data, as a plain object, including the apip data.

Reply:
@jsconan the link you've suggested will return the test itemref from the testmap (occurance, id, position, etc.). 
From the test runner I can imagine we can also add a `getItemRunner` or `getCurrentItemRunner` method.